doc/qos/SONiC_ROCEv2_support.md

```diff
@@ -964,6 +964,13 @@ config qos classifier clear --defaults
 - Ports created after moving to lossless mode should not be usable until next reboot.
 - While next bootup, All new ports are applied with lossless profiles based on "buffer init"/"roce enable" configuration.
 - All configurations removed before on port delete as part of DPB and set with NULL oid for PG and Q buffer profiles on that port.
+- CLI warning should be given to user in while giving DPB in lossless mode
+
+```
+sonic# configure terminal
+sonic(config)#interface breakout  port 1/1 mode 4x10G
+Warning: Switch in "lossless" mode, Ports created can be used only after reboot
+```

```

Comment:
Can we reboot the switch as part of the command ?

Reply:
breakout per port. For every individual break port port, doing the reboot internally is not best user experience.